src/app/assessments/assessments.module.ts

```diff
@@ -8,7 +8,6 @@ import { AssessmentsService } from './assessments.service';
 import { AssessmentsLayoutComponent } from './assessments-layout.component';
 import { AssessmentsDashboardComponent } from './assessments-dashboard/assessments-dashboard.component';
 import { AssessmentsDashboardService } from './assessments-dashboard/assessments-dashboard.service';
-import { GlobalModule } from 'app/global/global.module';
```

Comment:
npm run build:aot:prod complained about non relative path global module imports. not sure why, but I changed since it was only in a few places